source/read-write-concern/tests/operation/default-write-concern-3.4.yml

```diff
@@ -0,0 +1,197 @@
+# Test that setting a default write concern does not add a write concern
+# to the command sent over the wire.
+# Test operations that require 3.4+ server, where all commands started
+# to accept a write concern.
+
+data:
+  - {_id: 1, x: 11}
+  - {_id: 2, x: 22}
+collection_name: &collection_name default_write_concern_coll
+database_name: &database_name default_write_concern_db
+other_collection_name: &other_collection_name other_collection_name
+
+runOn:
+    - minServerVersion: "3.4"
+
+tests:
+  - description: Aggregate with $out omits default write concern
+    clientOptions: {writeConcern: {}}
+    operations:
+      - object: collection
+        databaseOptions: {writeConcern: {}}
+        collectionOptions: {writeConcern: {}}
+        name: aggregate
+        arguments:
+          pipeline: &out_pipeline
+            - $match: {_id: {$gt: 1}}
+            - $out: *other_collection_name
+          batchSize: 2
+    outcome:
+      collection:
+        name: *other_collection_name
+        data:
+          - {_id: 2, x: 22}
+    expectations:
+      - command_started_event:
+          command:
+            aggregate: *collection_name
+            pipeline: *out_pipeline
+            writeConcern: null
+  - description: RunCommand with a write command omits default write concern
+    clientOptions: {writeConcern: {}}
+    operations:
+      - object: database
+        databaseOptions: {writeConcern: {}}
+        name: runCommand
+        command_name: create
+        arguments:
+          command:
+            create: test_collection
+    expectations:
+      - command_started_event:
+          command:
+            create: test_collection
+            writeConcern: null
+  - description: CreateIndex and dropIndex omits default write concern 
+    clientOptions: {writeConcern: {}}
+    operations:
+      - object: collection
+        databaseOptions: {writeConcern: {}}
+        collectionOptions: {writeConcern: {}}
+        name: createIndex
+        arguments:
+          keys: {x: 1}
+      - object: collection
+        databaseOptions: {writeConcern: {}}
+        collectionOptions: {writeConcern: {}}
+        name: dropIndex
+        arguments:
+          name: x_1
+    expectations:
+      - command_started_event:
+          command:
+            createIndexes: *collection_name
+            indexes:
+              - name: x_1
+                key: {x: 1}
+            writeConcern: null
+      - command_started_event:
+          command:
+            dropIndexes: *collection_name
+            index: x_1
+            writeConcern: null
+  - description: MapReduce omits default write concern
+    clientOptions: {writeConcern: {}}
+    operations:
+      - name: mapReduce
+        object: collection
+        databaseOptions: {writeConcern: {}}
+        collectionOptions: {writeConcern: {}}
+        arguments:
+          map:
+            $code: 'function inc() { return emit(0, this.x + 1) }'
+          reduce:
+            $code: >-
```

Comment:
Is `>-` syntax for treating the value as a string and trimming any leading/trailing whitespace?

Reply:
According to https://yaml-multiline.info/ says `>` folds lines with single spaces, and the `-` means to add no newline at the end.

Regardless, I changed it back to follow suit with how it is specified in the mapReduce.yml. I regret ever passing these files through a YML formatter...